EMS/core-bundle/src/Command/Revision/DeleteCommand.php

```diff
@@ -101,9 +115,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return parent::EXECUTE_SUCCESS;
     }
 
-    private function deleteByQuery(): int
+    private function deleteByQuery(?string $queryJson): int
```

Comment:
can never be null you already checked

Reply:
I did not checked, but agree code can be better ,see last commit